RadarSDK/RadarLocationManager.m

```diff
@@ -94,6 +95,11 @@ - (instancetype)init {
         _lowPowerLocationManager.allowsBackgroundLocationUpdates = [RadarUtils locationBackgroundMode];
 
         _permissionsHelper = [RadarPermissionsHelper new];
+
+        // If not testing, set _notificationCenter to the currentNotificationCenter
+        if (![[NSProcessInfo processInfo] environment][@"XCTestConfigurationFilePath"]) {
+            _notificationCenter = [UNUserNotificationCenter currentNotificationCenter];
+        }
```

Comment:
I tried fairly hard to get a mock `UNUserNotificationCenter` going, but this ended up being what worked best.

Reply:
This is needed so that tests can work?

Reply:
Yes exactly. Without this, the tests error in trying to get the `currentNotificationCenter`

Reply:
I wonder why the old call site didn't to do this 🤔

Reply:
The notification codepath wasn't hit in the tests prior to enabling it in `trackOnce`